src/commcare_cloud/ansible/roles/elasticsearch/templates/monit.elasticsearch.j2

```diff
@@ -0,0 +1,6 @@
+check file elasticsearch with path {{ elasticsearch_data_dir }}/logs/icds-2.0.log
```

Comment:
Watch out for env-specific values! This took me hours to debug today, and luckily only affected staging—could have been a lot worse: https://github.com/dimagi/commcare-cloud/pull/3771. I know everyone's careful and we'll always make some mistakes, but we could all use a reminder now and then :)